build.gradle.kts

```diff
@@ -25,9 +25,6 @@ kotlin {
     }
 
     jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = "11"
```

Comment:
Is the intention to use whatever is the default right now or specifically target version 8? In the latter case we should probably be explicit about it.

Reply:
I made it explicit. I'm using the default so I only want that, but for libraries it's best to be explicit.